spotbugs/build.gradle

```diff
@@ -145,25 +145,16 @@ eclipse.classpath.file {
     }
 }
 
-def copyLibsForEclipse = tasks.register('copyLibsForEclipse', Copy) {
```

Comment:
This makes Eclipse build unhappy. Since there are no libraries available for Eclipse, I can't compile / run SpotBugs from Eclipse anymore.